File: Java/UberBlack.java
package Java;

import java.util.ArrayList;
import java.util.Map;

class UberBlack extends Car {
    Map<String,Map<String,Integer>> typeCarAccepted;
    ArrayList<String> seatsMaterial;


    public UberBlack(String licence,Account driver,Map<String,Map<String,Integer>> typeCarAccepted,ArrayList<String> seatsMaterial){
        super(licence, driver);
        this.typeCarAccepted =typeCarAccepted;
        this.seatsMaterial = seatsMaterial;
        
        
    }
    


}

File: Python/UberBlack.py
from car import Car

class UberBlack(Car):
    """
    UberBlack Class
    """
    typeCarAccepted  = []
    seatsMaterial    = []
    def __init__(self,license,driver,typeCarAccepted,seatsMaterial):
        super.__init__(license,driver)
        self.typeCarAccepted = typeCarAccepted
        self.seatsMaterial = seatsMaterial
